include/ensmallen_bits/moead/moead.hpp

```diff
@@ -72,8 +74,8 @@ class MOEAD {
    * @param upperBound The upper bound on each variable of a member
    *    of the variable space.
    */
-  MOEAD(const size_t populationSize = 150,
-        const size_t maxGenerations = 300,
+  MOEAD(const size_t populationSize = 300,
```

Comment:
![image](https://user-images.githubusercontent.com/45007169/122586740-a7848700-d07a-11eb-9486-77d2257eb429.png)

This is because the binomial coefficient value (H) should equate to the size of the population. This algorithm is restrictive in that way. You can read Section IV (If I recall correctly) of the cited paper to know more. To ensure this, you can find I've added an ```if``` statement inside ```Generate``` method to catch a mismatch. 

The main point is, there is only a certain number of points that can be generated uniformly given the number of partitions(gaps between points) and the number of objectives.